src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  let married;
+
+  if (person.isMarried) {
+    married = `${person.partnerName} is my ${person.sex === 'f' ? 'husband' : 'wife'}`;
+  } else {
+    married = `I am not married`;
```

Comment:
Let's use a ternary operator here to make your code cleaner
![image](https://github.com/mate-academy/react_person/assets/96409563/648e3a6a-2ce5-4620-bafc-c6cf187963b1)